### PR TITLE
Feature/restaurant page

### PR DESCRIPTION
**Pull request by damiecode**
Thank you for taking the time to review my code. Work-done
- [x] Setup project with Webpack 
- [x] Create a dummy page for the restaurant view 
- [x] Add entire contents using javaScript
- [x] Add tab-switching logic for the page and put all tabs in different modules.

### CODE REVIEW

File: dist/css/main.css [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+#menu, #contact {
+  background: #000;
+}
+
+.parallax-section {
+  background-attachment: fixed !important;
+  background-size: cover !important;
+}
+
+/* start navigation */
```
Comment (bellom) [inline dist/css/main.css]: It’s better to delete commented code. If you need to review a previous version of your code, use git history, your code is clear enough to understand 🙂 ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

Kindly remove all in this file.

File: dist/css/main.css [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
+  font-weight: bold;
+  letter-spacing: 2px;
+}
+.navbar-default .navbar-nav li a {
+  font-size: 12px;
+  letter-spacing: 1px;
+  color: #fff;
+}
+
+/* home section */
```
Comment (bellom) [inline dist/css/main.css]: It’s better to delete commented code. If you need to review a previous version of your code, use git history, your code is clear enough to understand 🙂 ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

File: package.json [hunk 3]
```diff
@@ -0,0 +1,30 @@
+{
+  "name": "Restaurant-Page",
+  "version": "1.0.0",
+  "description": "> ​  we are going to be using only JavaScript to generate the entire contents of a restaurant page!.",
+  "main": "index.js",
+  "scripts": {
```
Comment (bellom) [inline package.json:6]: We should be able to run common commands like `build`, `development` should be in `package.json` inside \`scripts'. Here is a basic example ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/scripts_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622983/)) in our own case here, we are missing it so I can't run `npm run build ` to generate `dist folder`, I should be able to run `npm run build` to generate `dist folder` at my end, currently using the one created by you, it shouldn't be like that.
kindly add these two scripts `build` and `development`

File: package.json [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+  },
+  "homepage": "https://github.com/damiecode/Restaurant-Page#readme",
+  "devDependencies": {
+    "css-loader": "^3.4.2",
+    "eslint": "^6.8.0",
+    "eslint-config-airbnb-base": "^14.0.0",
+    "eslint-plugin-import": "^2.20.0",
+    "file-loader": "^5.0.2",
+    "style-loader": "^1.1.3",
+    "webpack": "^4.41.5",
```
Comment (bellom) [inline package.json:29]: We should have all necessarily used packages that should be included in `package.json` like `webpack`, `webpack-cli`, `webpack-dev-server` etc. Also, those packages should be added as **devDependencies** ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/devdep_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622983/)) **in our case we are missing `webpack-dev-server`**
Kindly add  `webpack-dev-server`.

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @damiecode  :clap: 

### Awesome Design :+1: 

#### You have done wonderfully well, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 
 
Below are part of the checklist for this project, kindly work on these few things listed below;
Kindly my comments below - Just add the needed `script` and `devDependencies` and let move to the next project.

**Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....**

Best,

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Review by bellom — APPROVED** [review]
## Hi @damiecode  :clap: 

**Great work making all the necessary corrections from the previous review.**

## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit project completion form.
Good luck in the next project :slightly_smiling_face:_ 

Best,

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Comment by damiecode** [comment]
Thank you @bellom

**Pull request merged**